buildconfig/pygame-stubs/font.pyi

```diff
@@ -0,0 +1,29 @@
+from typing import List, Optional, Union, Tuple
+
+from pygame.color import Color
+from pygame.surface import Surface
+
+_ColorValue = Union[Color, Tuple[int, int, int], List[int], int, Tuple[int, int, int, int]]
+
+def init() -> None: ...
+def quit() -> None: ...
+def get_init() -> bool: ...
+def get_default_font() -> str: ...
+def get_fonts() -> List[str]: ...
+def match_font(name: str, bold: Optional[bool]=False, italic: Optional[bool]=False) -> str: ...
+def SysFont(name: str, size: int, bold: Optional[bool]=False, italic: Optional[bool]=False) -> Font: ...
+class Font(object):
+    def __init__(self, name: Union[str, None], size: int) -> None: ...
+    def render(self, text: str, antialias: bool, color: _ColorValue, background: Optional[_ColorValue]=None) -> Surface:...
+    def size(self, text: str) -> Tuple[int, int]: ...
+    def set_underline(self, value: bool) -> None: ...
+    def get_underline(self) -> bool: ...
+    def set_bold(self, value: bool) -> None: ...
+    def get_bold(self) -> bool: ...
+    def set_italic(self, value: bool) -> None: ...
```

Comment:
font.c@font_set_italic does not use a keyword argument.

Reply:
Had to write something